app/manage-breast-screening/2025/11/ending-breast-screening-appointments/index.md

```diff
@@ -0,0 +1,133 @@
+* they fail the ID check at reception
+* the clinic can't support their needs
+* a technical issue at the clinic
+
+For each of these, admin teams would view the appointment then select 'Cancel or reschedule appointment'.
+
+![Screenshot of the appointment view including a a link to cancel or reschedule](cancel-appointment-link.png)
+
+They will first be asked to explain why. The options for the second question cover whether a participant is opting out of screening (either temporarily or permanently), and whether the appointment needs to be rescheduled.
+
+![Screenshot of a form with reasons for cancelling and options for rescheduling the appointment](cancel-or-reschedule.png)
+
+### Working to current constraints
+
+If rescheduling is required, we're offering the option to rearrange it within or outside a 6-week window.
+
+![Screenshot of a form providing options for rescheduling the appointment](reschedule-info.png)
+
+This reflects the limitations of the current booking system. We're using this as a starting point, while working with colleagues rebuilding the screening invitation system to ensure any options provided here are in line with future functionality. 
+
+## 2. Stopping the appointment after it starts
+
+Once the mammographer conducting the screening has actively begun the appointment workflow, two statuses are possible; `Attended not screened` (ANS) and `Partial mammogram`.
+
+The partial mammogram scenario covers any instance where at least one X-ray has been taken, but the appointment could not be concluded. This is a more complicated journey that we are tackling separately.
+
+The reasons for a participant to be marked as ANS include: 
+
+* they decide to opt out
+* they don't understand instructions given to them
+* they fail the ID check in the mammogram room
+* they've had a total bilateral mastectomy
+* there are technical issues with equipment before any images are taken
+
+At each stage of the workflow, an 'Appointment cannot proceed' link is available. When users select this they are asked for a reason. This list is very similar to the 'Cancel appointment' options, but has been adapted to include things that can only occur once the appointment has actually begun.
+
+![Screenshot of a form with reasons for stopping the appointment after it has begun and options for rescheduling](appointment-cannot-proceed.png)
+
+As well as actively stopping an appointment, users may reach an end point based on other information they enter into the service.
+
+### Recent mammograms
+
+For example, if the participant tells them about a mammogram that took place within the last 6 months, they would be advised not to go ahead.
+
+![Screenshot of a conditional page that appears when a recent mammogram has been recorded](recent-mammogram.png)
+
+### Withdrawn consent
+
+If the participant does not consent to mammograms of breast implants, users would be given options for how to end the appointment.
```

Comment:
Could preface by saying that additional consent must be sought in x scenarios...